plugin/src/main/java/io/spine/tools/gradle/ProjectDependencyTarget.java

```diff
@@ -20,32 +20,56 @@
 
 package io.spine.tools.gradle;
 
+import com.google.common.collect.ImmutableMap;
 import org.gradle.api.Project;
+import org.gradle.api.artifacts.Configuration;
+import org.gradle.api.artifacts.ConfigurationContainer;
 import org.gradle.api.artifacts.dsl.DependencyHandler;
 
 import static com.google.common.base.Preconditions.checkNotNull;
+import static io.spine.tools.gradle.ConfigurationName.RUNTIME_CLASSPATH;
+import static io.spine.tools.gradle.ConfigurationName.TEST_RUNTIME_CLASSPATH;
 
 /**
  * A {@link DependencyTarget} implemented on top of a {@link DependencyHandler}  of a project.
  */
 public final class ProjectDependencyTarget implements DependencyTarget {
 
     private final DependencyHandler dependencies;
+    private final ConfigurationContainer configurations;
 
-    private ProjectDependencyTarget(DependencyHandler dependencies) {
+    private ProjectDependencyTarget(DependencyHandler dependencies,
+                                    ConfigurationContainer configurations) {
         this.dependencies = dependencies;
+        this.configurations = configurations;
     }
 
     /**
      * Creates a new instance of {@code ProjectDependencyTarget} for the given project.
      */
     public static ProjectDependencyTarget from(Project project) {
         checkNotNull(project);
-        return new ProjectDependencyTarget(project.getDependencies());
+        return new ProjectDependencyTarget(project.getDependencies(), project.getConfigurations());
     }
 
     @Override
     public void depend(String configurationName, String notation) {
         dependencies.add(configurationName, notation);
     }
+
+    @Override
+    public void exclude(Dependency dependency) {
+        Configuration mainConfig = configurations.getByName(RUNTIME_CLASSPATH.getValue());
+        exclude(mainConfig, dependency);
+
+        Configuration testConfig = configurations.getByName(TEST_RUNTIME_CLASSPATH.getValue());
+        exclude(testConfig, dependency);
+    }
+
+    private static void exclude(Configuration configuration, Dependency module) {
+        configuration.exclude(ImmutableMap.of(
+                "group", module.groupId(),
```

Comment:
I think the literals used here deserve to be documented. Please have a Javadoc for this method.